core/frontend/src/extension/Extension.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Extensions
+ */
+
+/**
+ * @beta
+ */
+export enum ActivationEvent {
+  onStartup = "onStartup",
+}
+
+/**
+ * @beta
+ */
+export type ResolveFunc = () => Promise<any>;
+
+/** Defines the format of an Extension manifest
+ * @beta
+*/
+export interface ExtensionManifest {
+  /** The extension name */
+  readonly name: string;
+  /** The extension display name */
+  readonly displayName?: string;
+  /** The extension version */
+  readonly version: string;
+  /** The extension description */
+  readonly description?: string;
+  /** The extension author */
+  readonly author: string;
```

Comment:
Anything specific expected here? An email address or other contact info?

Reply:
This will be from package.json in most cases, which would follow those [conventions](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors). So, if we start to care about the author (use in code), assuming that this is a string could be misleading. @calebmshafer @aruniverse I feel like we remove this for now. Thoughts?

Reply:
removed